src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -1478,19 +1479,11 @@ export function getMultiSelectedEditorContexts(editorContext: IEditorCommandsCon
 	// Check editors selected in the group (tabs)
 	else {
 		const group = editorContext ? editorGroupService.getGroup(editorContext.groupId) : editorGroupService.activeGroup;
-		if (group) {
-			const selectedEditors: EditorInput[] = [];
-			// If context provides an editor index, use it
-			if (editorContext && editorContext.editorIndex !== undefined) {
-				const editor = group?.getEditorByIndex(editorContext.editorIndex);
-				if (editor && group.isSelected(editor)) {
-					selectedEditors.push(...group.selectedEditors);
-				}
-			} else {
-				selectedEditors.push(...group.selectedEditors);
-			}
-			if (selectedEditors.length > 1) {
-				return selectedEditors.map(se => ({ groupId: group.id, editorIndex: group.getIndexOfEditor(se) }));
+		const editor = editorContext && editorContext.editorIndex !== undefined ? group?.getEditorByIndex(editorContext.editorIndex) : group?.activeEditor;
```

Comment:
Is `editorContext.editorIndex` ever going to be `undefined`?

Reply:
same as above